test/sparse/sparsevector.jl

```diff
@@ -1,4 +1,4 @@
-# This file is a part of Julia. License is MIT: http://julialang.org/license
+<# This file is a part of Julia. License is MIT: http://julialang.org/license
```

Comment:
https://github.com/JuliaLang/julia/pull/19721 also means rebase needed for end-of-deprecated.jl